.gitignore

```diff
@@ -1288,3 +1288,509 @@ node_modules/yallist/README.md
 node_modules/yallist/yallist.js
 .DS_Store
 dist/views/views/index.html
+node_modules/@popperjs/core/index.d.ts
+node_modules/@popperjs/core/LICENSE.md
+node_modules/@popperjs/core/package.json
+node_modules/@popperjs/core/README.md
+node_modules/@popperjs/core/dist/cjs/enums.js
+node_modules/@popperjs/core/dist/cjs/enums.js.flow
+node_modules/@popperjs/core/dist/cjs/enums.js.map
+node_modules/@popperjs/core/dist/cjs/popper-base.js
+node_modules/@popperjs/core/dist/cjs/popper-base.js.flow
+node_modules/@popperjs/core/dist/cjs/popper-base.js.map
+node_modules/@popperjs/core/dist/cjs/popper-lite.js
+node_modules/@popperjs/core/dist/cjs/popper-lite.js.flow
+node_modules/@popperjs/core/dist/cjs/popper-lite.js.map
+node_modules/@popperjs/core/dist/cjs/popper.js
+node_modules/@popperjs/core/dist/cjs/popper.js.flow
+node_modules/@popperjs/core/dist/cjs/popper.js.map
+node_modules/@popperjs/core/dist/esm/createPopper.js
+node_modules/@popperjs/core/dist/esm/enums.js
+node_modules/@popperjs/core/dist/esm/index.js
+node_modules/@popperjs/core/dist/esm/popper-base.js
+node_modules/@popperjs/core/dist/esm/popper-lite.js
+node_modules/@popperjs/core/dist/esm/popper.js
+node_modules/@popperjs/core/dist/esm/types.js
+node_modules/@popperjs/core/dist/esm/dom-utils/contains.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getBoundingClientRect.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getClippingRect.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getCompositeRect.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getComputedStyle.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getDocumentElement.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getDocumentRect.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getHTMLElementScroll.js
+node_modules/@popperjs/core/dist/esm/dom-utils/getLayoutRect.js
```

Comment:
😮  wow! These are all caught by just `node_modules/` I think!

Reply:
@jkelso662 Seems to be, I'll update this to exclude everything within node_modules/